wandb/sdk/lib/module.py

```diff
@@ -15,11 +15,11 @@ def set_global(
 ):
     if run:
         wandb.run = run
-    if config:
+    if config is not None:
```

Comment:
I dont get why this is needed.

Reply:
It is not needed for config now since we use the real Config, but for summary we use a DummyDict in noop mode which evaluates to False if empty, so we need to distinguish between None and empty dict.